src/collapse/bpftrace.rs

```diff
@@ -2,103 +2,123 @@ use std::io;
 use std::iter::Peekable;
 use std::mem;
 
-#[derive(Debug)]
-enum State {
-    NotInStack,
-    InStack(Vec<String>),
+use crate::collapse::Frontend;
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Hash)]
+pub struct Bpftrace {
+    state: State,
+}
+
+impl Bpftrace {
+    pub fn new() -> Self {
```

Comment:
I'd probably cut this and just have `Default`

Reply:
👍